runtime/jcl/common/mgmtmemory.c

```diff
@@ -702,6 +703,59 @@ Java_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificati
 				return;
 			}
 
+			/* error tracepoint for the cases
+				the value of init or max is negative but not -1; or
+				the value of used or committed is negative; or
+				used is greater than the value of committed; or
+				committed is greater than the value of max if max is not -1.
+			*/
```

Comment:
Please fix this comment; it's not formatted properly and it refers to impossible conditions (that have since been removed).